node_modules/col-whiteboards/lib/api.js

```diff
@@ -755,7 +756,24 @@ var whiteboardAddActivity = function(socket, elements, callback) {
       AnalyticsAPI.track(socket.ctx.user, 'Add whiteboard element', AnalyticsAPI.getWhiteboardElementProperties(element, socket.whiteboard, elements.length));
     });
 
-    return callback();
+    async.each(elements, function(element, done) {
```

Comment:
That would be the right way to do them serially, so might be the safest thing to do here.

A potentially better would be to allow `ActivitiesAPI.createActivity` to process multiple activities at once, but that might be out of scope for this PR.